### PR TITLE
[BCAMDT-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by lutechongo**
Added navigation links to product page and contact form.

### CODE REVIEW

File: submissions/luteChongo/html/product-page.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h1 style="text-align: center; font-family: fantasy;"><u>ZANI STORES</u></h1>
 		<div class="row">
 			<nav>
 				<li><a href="#">Product</a></li>
-				<li><a href="file:///C:/Users/morta/Desktop/Dufuna-CodeCamp-21/submissions/luteChongo/html-forms/contact-form-page.html">Contact Form</a></li>
+				<li><a href="file:///C:/Users/morta/Desktop/Dufuna-CodeCamp-21/Submissions/luteChongo/html-forms/contact-form-page.html">Contact Form</a></li>
```
Comment (Ireti203) [inline submissions/luteChongo/html/product-page.html:22]: This link is absolute and will only work at your end i.e I don't have this exact directory so it wont work for me. You can simply do '../html-forms/contact-form-page.html'

Also you ought to have the links to both the product page and contact page on both pages.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/luteChongo/html/product-page.html (hunk 1 above)
Comment (lutechongo) [reply submissions/luteChongo/html/product-page.html:22]: I have done so but it is saying that the file cant be accessed. I have also tried "../html/product-page.html" but it is saying the same thing.
Screenshot (22)

**Review by Ireti203 — COMMENTED** [review]
Since you are on the product-page.html page itself, simply do "product-page.html" directly

**Comment by lutechongo** [comment]
> Since you are on the product-page.html page itself, simply do "product-page.html" directly

I have done so. But once I put them back under the respective folders in submissions and try the links you've told me to put, it says files cant be accessed. 

May we have a google meet on Monday at a time you prefer so I show you what I mean?

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**